src/commands/quote.ts

```diff
@@ -4,7 +4,9 @@ import { MotivationalDef } from '../APIs/quote-def';
 
 export const quote: CommandDef = {
   prefix: 'quote',
-  description: 'Returns a quote from the FCC motivational quotes file',
+  description:
+    "Returns a quote fetched from freeCodeCamp's motivational quotes API",
```

Comment:
To clarify, it's fetched from the FCC `motivational-quotes.json` file, but the fetch uses the **GitHub** API. :)